src/content/tutorials/observability/multi-tenancy/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Multi-tenancy
+description: Guides explaining how to configure multi-tenancy in the Observability Platform.
+weight: 20
+menu:
+  principal:
+    parent: tutorials-observability
+    identifier: tutorials-observability-multitenancy
+last_review_date: 2024-12-12
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - How to isolate data into the observability platform?
+  - How to create grafana organizations?
+---
```

Comment:
Should we already add this, if it's empty at this time? I'm not confident adding empty docs into the tree, that is just a confusing experience, if you found what you searched for and then it's empty. WDYT?

Reply:
It's not empty though, it's a page that will redirect to the creating grafana organization page but I can try to remove it

Reply:
But are you filling it later? It is fine to leave it empty to pass the CI meanwhile you add the content

Reply:
Well is that not how summary pages work? They are empty and the tool creates the links?

Reply:
ah wait, you are right, this is just the head of section

Reply:
My bad sorry.